### PR TITLE
Add AuthenticationTokenWebhook flag

### PR DESCRIPTION
**Pull request by ihoegen**
Addresses #5212

### CODE REVIEW

**Comment by mikesplain** [comment]
Great, thanks for this @ihoegen!
And no need need for ok-to-test!
/lgtm

File: pkg/apis/kops/componentconfig.go [hunk 1]
```diff
@@ -296,6 +296,8 @@ type KubeAPIServerConfig struct {
 	AuditLogMaxSize *int32 `json:"auditLogMaxSize,omitempty" flag:"audit-log-maxsize"`
 	// AuditPolicyFile is the full path to a advanced audit configuration file a.g. /srv/kubernetes/audit.conf
 	AuditPolicyFile string `json:"auditPolicyFile,omitempty" flag:"audit-policy-file"`
+	// AuthenticationTokenWebhook enables bearer token authentication on kubelet.
+	AuthenticationTokenWebhook *bool `json:"authenticationTokenWebhook,omitempty" flag:"authentication-mode-webhook"`
```
Comment (justinsb) [inline pkg/apis/kops/componentconfig.go]: The flag name should be `authentication-token-webhook`, I believe

Comment (ihoegen) [reply pkg/apis/kops/componentconfig.go]: You're right, I must've blanked. I'll fix it real quick

**Comment by ihoegen** [comment]
@justinsb Should be good now, I regenerated the apimachinery too, just in case

**Comment by justinsb** [comment]
Awesome - thanks @ihoegen 

/approve
/lgtm

**Comment by samueltbrown** [comment]
Hello!  I just tried to use this feature but I found that the flag was placed in the kube-apiserver section instead of the kubelet section and authentication-token-webhook does not seem to be a valid flag according to [api server docs](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).  Should this have instead been added to the kubelet configuration section per the [kubelet docs](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)?

**Comment by ihoegen** [comment]
Just opened a PR to fix that https://github.com/kubernetes/kops/pull/5508, thanks

**Comment by rafilkmp3** [comment]
How i can create a cluster from command line with this enabled ?

```
unknown flag: --authentication-token-webhook
```

**Pull request merged**